### PR TITLE
Updated what's new with changes since Sept 20, 2019

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request updates the [What's New](https://devdocs.magento.com/whats-new.html) page with the documentation changes published since the last update on Sept 20.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

**Comment by meker12** [comment]
> I do not see an entry for PR #5522 merged yesterday.

Will include in the next version of What's New.

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,39 @@ description: |
...
+  type: Major update
+  date: October 1, 2019
+  link: https://github.com/magento/devdocs/pull/5489
+- description: Added the [RemainingCharacters widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-remaining-characters.html)
+    topic to the _Javascript Developer Guide_.
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: September 24, 2019
+  link: https://github.com/magento/devdocs/pull/5452
+- description: Added example [introspection queries](https://devdocs.magento.com/guides/v2.3/graphql/queries.html)
```
Comment (dshevtsov) [inline _data/whats-new.yml]: This entry is a duplicate of the previous entry.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**